src/app/manager-dashboard/manager-dashboard.component.ts

```diff
@@ -84,4 +91,10 @@ export class ManagerDashboardComponent implements OnInit {
       }, error => this.planetMessageService.showAlert('An error occurred please try again.'));
   }
 
+  checkRequestAcceptedOrNot() {
+    if (this.userService.getConfig().registrationRequest === 'accepted') {
```

Comment:
As we do not sync configuration back to community right now we should look for it on parent. My previous comment mentioned **Later** we will not require to look parent. But right now this feature will not work.